### PR TITLE
Bugfix/twcookie 11/twookie 20 dev review

### PR DESCRIPTION
**Pull request by Martin-Welte**
## TLDR; 

* minor refactoring: mainly typos & deletion of unnecessary docblocks

## What to test & how to test 

* nothing to test. no major code changes

## solves Issues
* no issues solved

## Updated
- Added link to official GDPR website.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,3 +19,5 @@ You will find the steps required for the usage in the [`docs`](/docs/) folder.
 
 ## Contributors
 - [Towa Digital Agency](https://www.towa.at)
+- [Martin-Welte](https://github.com/Martin-Welte)
+- [szepeviktor](https://github.com/szepeviktor)
```
Comment (Martin-Welte) [inline README.md:24]: @szepeviktor is it ok for you if I mention you?

Comment (szepeviktor) [reply README.md:24]: Yes, Martin.
Danke schön.

Comment (Martin-Welte) [inline README.md]: @szepeviktor is it ok if I mention you?

**Review by szepeviktor — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 # Towa GDPR Plugin
 
 This Plugin is a cookie notice plugin conforming with the GDPR regulation, implemented by the EU.
```
Comment (szepeviktor) [inline README.md]: @Martin-Welte Could you link "GDPR regulation"?
https://eur-lex.europa.eu/eli/reg/2016/679/oj (down at the moment, actually the whole EU is _quiet down_ atm)

**Review by d4ny0 — APPROVED** [review]

**Pull request merged**